### DISCUSSION TITLE
New privatebin setup, unable to save

### DISCUSSION THREAD

**Discussion by ashok-raj** [PrivateBin/PrivateBin#1100] [General]
Hi

I just setup privatebin, but when i create a new pastefile and press "send" on the top right, it always gives me the error 

"Could not create paste: Error saving paste. Sorry."

What should I check?

**Comment by r4sas** [comment]
Logs.

**Comment by ppfeufer** [comment]
If you installed via Docker, have a look here » https://ppfeufer.de/privatebin-your-self-hosted-pastebin-instance/ « and scroll down to "Fix Permissions".

**Reply by ghost** [reply]
> If you installed via Docker, have a look here » https://ppfeufer.de/privatebin-your-self-hosted-pastebin-instance/ « and scroll down to "Fix Permissions".

this link to this guide fixed my issue located below, hope this helps someone else. i will also link to my issue and mark resolved.

https://github.com/orgs/PrivateBin/discussions/1098#discussion-5287803

**Reply by ashok-raj** [reply]
I had to fixup the permissions.. things work normally now!.... Thanks everyone.

**Comment by ashok-raj** [comment]
Permissionf or PrivateBin/data had to be fixed up..
